frontend/public/components/factory/list-page.jsx

```diff
@@ -29,7 +28,7 @@ export const CompactExpandButtons = ({expand = false, onExpandChange = _.noop})
   </label>
 </div>;
 
-/** @type {React.SFC<{autofocus?: boolean, disabled?: boolean, label: string, onChange: React.ChangeEventHandler<any>, defaultValue: string}}>} */
+/** @type {React.SFC<{autoFocus?: boolean, disabled?: boolean, label: string, onChange: React.ChangeEventHandler<any>, defaultValue: string}}>} */
```

Comment:
Prefer `autofocus` since it's [one word](https://www.dictionary.com/browse/autofocus).

Reply:
This was actually a bug from one of my previous PRs. The prop was implemented as `autoFocus` previously and I misspelled it. In the React type definitions for HTML attributes, it is defined as `autoFocus`.

Reply:
Ah OK, disregard my comment :)